test/conformance/runtime/container_test.go

```diff
@@ -80,26 +80,23 @@ func TestShouldNotContainerConstraints(t *testing.T) {
 	clients := test.Setup(t)
 
 	testCases := []struct {
-		name    string
-		options func(s *v1.Service)
+		name            string
+		options         func(s *v1.Service)
+		assertIfNoError func(t *testing.T, s *v1.Service)
 	}{{
-		name: "TestPoststartHook",
+		name: "TestLifecycle",
 		options: func(s *v1.Service) {
 			lifecycleHandler := &corev1.ExecAction{
 				Command: []string{"/bin/sh", "-c", "echo Hello from the post start handler > /usr/share/message"},
 			}
 			s.Spec.Template.Spec.Containers[0].Lifecycle = &corev1.Lifecycle{
 				PostStart: &corev1.LifecycleHandler{Exec: lifecycleHandler},
+				PreStop:   &corev1.LifecycleHandler{Exec: lifecycleHandler},
```

Comment:
Should `PreStop` be a second test, to avoid having the test spuriously pass if it only handles `PreStart` and not `PreStop`?

Reply:
I can split it like before